app/assets/javascripts/tylium/modules/local_auto_save.js

```diff
@@ -45,7 +56,15 @@ document.addEventListener("turbolinks:load", function() {
 
     function getData(formInputs) {
       var reducer = function(hash, input) {
-        hash[input.name] = input.value;
+        // Check if name is an array, i.e. checkboxes
+        if (input.name.slice(-2) == "[]") {
```

Comment:
What are we saving checkboxes for?

Reply:
@aapomm Checkboxes are needed for the assignee ids in cards#form
![image](https://user-images.githubusercontent.com/18374402/80453898-bb8f9580-895b-11ea-8469-782afce7a822.png)